python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -1102,9 +1166,36 @@ def _sensor(context):
             instance=context.instance,
             cursor=cursor,
             run_tags=run_tags,
+            auto_observe=False,
```

Comment:
can we just remove this boolean and just have this always be True? this sensor will be removed imminently anyway, and I don't see a huge benefit to turning off the behavior for one release

Reply:
asset selections involving source assets are kinda gnarly, and I think that unless we're really careful – aka add some tests and stuff – the asset reconciliation sensor will end up accidentally automatically observing some source assets that the user didn't expect to include.

if it's purely about saving effort, ripping out this parameter feels easier than adding those tests

Reply:
* by "ripping out this parameter" I mean "submitting a PR next week to rip out this parameter"

Reply:
sounds good